packages/components/src/components/editable/Cell.tsx

```diff
@@ -277,9 +271,23 @@ export class Cell extends React.PureComponent<Props, any> {
             return <LookupCell {...lookupProps} />;
         }
 
+        let renderer;
```

Comment:
It seems like a lot of the special handling introduced here is due to the "Alias" column that appears on Sample Sets. Was there explicit direction given to make sure that column works in this context? Could we re-use the `AliasInput` as written with some styling updates so we don't have to replicate logic?

Reply:
Same as above. I didn't want to add special handling directly in this general use case so relegated special cell value handling to the renderer.  I did not get any direction on if we want the comma separated vs alias input.  I went with comma separated to keep the context of the edit grid consistent with the display grid and details. But open to discuss with biologics team if we want this to be the AliasInput